package.json

```diff
@@ -60,23 +60,24 @@
   },
   "devDependencies": {
     "@ngtools/webpack": "~1.1.7",
```

Comment:
With this new set of packages I get unmet peer dependency warnings if `@ngtools/webpack` v1.1.8 is used, removing the `~` ensures that can't happen.